src/Tasks/Microsoft.NET.Build.Tasks/ResolvePublishAssemblies.cs

```diff
@@ -32,7 +33,9 @@ public class ResolvePublishAssemblies : TaskBase
 
         public bool PreserveCacheLayout { get; set; }
 
-        public string FilterProjectAssetsFile { get; set; }
+        public bool DoNotTrackPackageAsResolved { get; set; }
+
+        public string FilterProjFiles { get; set; }
```

Comment:
IIRC, you can pass semicolon delimited properties to array arguments of tasks and it does what we want. If so, this is not a reason to have manual splits in the code. cc @rainersigwald